SPECS-EXTENDED/CharLS/CharLS.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "CharLS-2.0.0.tar.gz": "528c6a3cc168a44e73f2890d8f4a35104a54d752eba3d6a643f050b72dd67cfa"
+  "CharLS-2.4.2.tar.gz": "d1c2c35664976f1e43fec7764d72755e6a50a80f38eca70fcc7553cad4fe19d9"
```

Comment:
Uploaded to blobstore:
- [PR-10532](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=646692&view=results)